lib/ui/bottombar/test_bottombar_navigation.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:farmsmart_flutter/data/bloc/article/ArticleListProvider.dart';
+import 'package:farmsmart_flutter/data/repositories/FlameLink.dart';
+import 'package:farmsmart_flutter/data/repositories/article/ArticleRepositoryInterface.dart';
+import 'package:farmsmart_flutter/data/repositories/article/implementation/ArticlesRepositoryFlamelink.dart';
+import 'package:farmsmart_flutter/ui/bottombar/tab_navigator.dart';
+import 'package:farmsmart_flutter/ui/discover/ArticleList.dart';
+import 'package:farmsmart_flutter/ui/playground/data/playground_datasource_impl.dart';
+import 'package:farmsmart_flutter/ui/playground/playground_view.dart';
+import 'package:farmsmart_flutter/utils/assets.dart';
+import 'package:flutter/material.dart';
+
+import 'bottom_navigation_bar_controller.dart';
+
+final cms =
```

Comment:
try to minimise the dependencies in tests.
Maybe keep it just to simple mocks.

Reply:
Well this class will be removed before the merge, because it's not a test itself I've just created this "test app" to test this widget while I was working on it. I will add this widget to the playground with mock data and mock tabs

Reply:
let´s remove before the PR else how does this get into dev? We should have a playground and widget test to demo its use, and that should remain.

Reply:
👍